Sources/JExtractSwift/Swift2JavaTranslator+Printing.swift

```diff
@@ -504,21 +440,24 @@ extension Swift2JavaTranslator {
        *
       \(decl.renderCommentSnippet ?? " *")
        */
-      public \(parentName.unqualifiedJavaTypeName)(SwiftArena arena, \(renderJavaParamDecls(decl, paramPassingStyle: .wrapper))) {
-        var mh$ = \(descClassIdentifier).HANDLE;
-        try {
+      public \(parentName.unqualifiedJavaTypeName)(\(renderJavaParamDecls(decl, paramPassingStyle: .wrapper)), SwiftArena arena) {
```

Comment:
I'm 90% sure that `arena` parameter at the end is more common practice.
Although I haven't been able to find any wrapper constructor examples,
* [SymbolLookup::libraryLookup(String,Arena)](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/foreign/SymbolLookup.html#libraryLookup(java.lang.String,java.lang.foreign.Arena))
* [Linker::upcallStub(MethodHandle,FunctionDescriptor,Arena,Linker.Option...)](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/foreign/Linker.html#upcallStub(java.lang.invoke.MethodHandle,java.lang.foreign.FunctionDescriptor,java.lang.foreign.Arena,java.lang.foreign.Linker.Option...)
)
* Generated function pointer binding `allocate` https://github.com/openjdk/jextract/blob/master/samples/libjimage/JImageFile.java#L65-L71

Reply:
Last sounds ok me, makes the calls more similar to their swift equivalents and only the arena added on.